src/vaadin-custom-field-mixin.html

```diff
@@ -122,13 +131,14 @@
       this.__labelId = `${this.constructor.is}-label-${uniqueId}`;
     }
 
-    /*
-     * @private
-     */
     focus() {
       this.inputs && this.inputs[0] && this.inputs[0].focus();
     }
 
+    /**
+     * @param {!Event | undefined} e
+     * @protected
```

Comment:
IMO this could be `@private` even though it is called from other file. Not a blocked.

Reply:
Updated to private.